web/frontend/src/components/slam/2d-renderer.vue

```diff
@@ -1,47 +1,48 @@
 <script setup lang="ts">
 
 import { $ref } from 'vue/macros';
-import { threeInstance, MouseRaycaster, MeshDiscardMaterial } from 'trzy';
+import { threeInstance, MouseRaycaster, MeshDiscardMaterial, GridHelper } from 'trzy';
 import { onMounted, onUnmounted, watch } from 'vue';
 import * as THREE from 'three';
 import { MapControls } from 'three/examples/jsm/controls/MapControls';
 import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader';
 import type { commonApi } from '@viamrobotics/sdk';
-import DestMarker from '../lib/destination-marker.txt?raw';
-import BaseMarker from '../lib/base-marker.txt?raw';
+import DestMarker from '@/lib/images/destination-marker.txt?raw';
+import BaseMarker from '@/lib/images/base-marker.txt?raw';
```

Comment:
Related to my other comment, I think the manual base64-ification might be unnecessary. My local build comes out exactly the same file count and size if I swap this with the png's directly.

```suggestion
import DestMarker from '@/lib/images/destination-marker.png';
import BaseMarker from '@/lib/images/base-marker.png';
```

Reply:
Yeah, agreed that this is nicer, and I think it works if you agree with my previous comment